RELEASING.md

```diff
@@ -45,8 +38,23 @@ You are done!
 [releases/list]:   https://github.com/lightbend/mima/releases
 [releases/new]:    https://github.com/lightbend/mima/releases/new
 
+[RELEASING.md]: https://raw.githubusercontent.com/lightbend/mima/master/RELEASING.md
 [repo1/list]: https://repo1.maven.org/maven2/com/typesafe/mima-core_2.12/0.5.0/
 [sbt-mima/view]: https://bintray.com/typesafe/sbt-plugins/sbt-mima-plugin/view
 [sonatype/guide]: https://central.sonatype.org/pages/releasing-the-deployment.html
 [sonatype/staging-repos]: https://oss.sonatype.org/#stagingRepositories
 [travis-ci]: https://travis-ci.org/lightbend/mima
+
+You are done!
+
+## Prerequisites
```

Comment:
A bit strange to close with the prerequisites :)

Reply:
Perhaps, but the motivation is to keep the signal-to-noise at the start of the document as low as possible.

Reply:
Oh, and the change here is I moved the URL refs up so they can be part of the c+p when creating the issue.